packages/filebrowser/src/listing.ts

```diff
@@ -1977,6 +1977,13 @@ export namespace DirListing {
         const checkboxWrapper = this.createCheckboxWrapperNode({
           alwaysVisible: true
         });
+        const checkboxInput = checkboxWrapper.querySelector(
+          'input[type="checkbox"]'
+        );
```

Comment:
This actually seems to be breaking things.

```ts
TypeError: Cannot read properties of null (reading 'querySelector')
    at Renderer.updateItemNode (listing.ts:2252:45)
    at listing.ts:817:16
    at Array.forEach (<anonymous>)
    at DirListing.onUpdateRequest (listing.ts:814:11)
    at DirListing.processMessage (index.es6.js:1222:1)
    at invokeHandler (index.es6.js:400:1)
    at sendMessage (index.es6.js:136:1)
    at runMessageLoop (index.es6.js:451:1)
```

![checkboxes-buggy](https://user-images.githubusercontent.com/5832902/210246047-25f00031-a340-4411-b6a1-08ffd2de2345.gif)

Reply:
Good catch, thanks. Fixed in https://github.com/jupyterlab/jupyterlab/pull/13682/commits/4b532287f77c9280eeb6d0ff22e69b30813c9dc1